test/test_other.py

```diff
@@ -7537,12 +7537,6 @@ def test(contents, expected, args=[], assert_returncode=0):  # noqa
     # test backwards compatibility
     test("Module['print']('print'); Module['printErr']('err'); ", 'print\nerr', ['-sEXPORTED_RUNTIME_METHODS=print,printErr'])
 
-  def test_warn_unexported_main(self):
-    WARNING = 'main() is in the input files, but "_main" is not in EXPORTED_FUNCTIONS, which means it may be eliminated as dead code. Export it if you want main() to run.'
-
-    proc = self.run_process([EMCC, test_file('hello_world.c'), '-sEXPORTED_FUNCTIONS=[]'], stderr=PIPE)
-    self.assertContained(WARNING, proc.stderr)
```

Comment:
Why is no longer needed?

Reply:
Its not that is no longer needed bu that it was the `llvm-nm` process that we ran on the input files which was responsible for reporting this error.

I'll see if I can resurrect this is a different form.

Reply:
I managed to keep this test alive.. but being a bit sneaky and adding `main` to EXPORT_IF_DEFINED in debug builds.